src/main/java/gregtech/common/metatileentities/electric/MetaTileEntityTransformer.java

```diff
@@ -138,4 +145,27 @@ protected boolean openGUIOnRightClick() {
     protected ModularUI createUI(EntityPlayer entityPlayer) {
         return null;
     }
+
+    @Override
+    public void addInformation(ItemStack stack, @Nullable World player, List<String> tooltip, boolean advanced) {
+        String lowerTierName = GTValues.VN[getTier() - 1];
+        String higherTierName = GTValues.VN[getTier()];
+        long lowerVoltage = energyContainer.getOutputVoltage();
+        long higherVoltage = energyContainer.getInputVoltage();
+        long lowerAmperage = energyContainer.getInputAmperage();
+        long higherAmperage = energyContainer.getOutputAmperage();
+
+        tooltip.add(I18n.format("gregtech.machine.transformer.tooltip"));
+        tooltip.add(I18n.format("gregtech.universal.tooltip.energy_storage_capacity", energyContainer.getEnergyCapacity()));
```

Comment:
energy storage capacity isn't needed there really
not sure about amperage in/out, too much text as far as i can say